react-app/src/library/services/vendor-implementations/jabra/jabra.ts

```diff
@@ -370,16 +372,29 @@ export default class JabraService extends VendorImplementation {
     this.changeConnectionStatus({ isConnected: true, isConnecting: false });
   }
 
+  async deviceHasPermissions (deviceLabel: string): Promise<boolean> {
+    const allowedHIDDevices = await (window.navigator as any).hid.getDevices();
+    let deviceFound;
+    allowedHIDDevices.forEach(device => {
+      if (deviceLabel.includes(device?.productName?.toLowerCase())) {
+        deviceFound = true;
+        return;
+      } else {
+        deviceFound = false;
+        return;
+      }
+    });
+    return deviceFound;
+  }
+
   async getPreviouslyConnectedDevice (deviceLabel: string): Promise<IDevice> {
```

Comment:
is there any chance this doesn't ever return? I'm wondering if we should at least log something after a given amount of time. If we bump into timeout issues, it would at least give us something to hassle jabra about.